lib/components/tabs.vue

```diff
@@ -146,28 +142,28 @@
              * Move to next tab
              */
             nextTab() {
-                this.setTab(this.currentTab, false, 1);
+                this.setTab(this.currentTab + 1, false, 1);
             },
 
             /**
              * Move to previous tab
              */
             previousTab() {
-                this.setTab(this.currentTab, false, -1);
+                this.setTab(this.currentTab - 1, false, -1);
```

Comment:
It did, but it couldn't handle wrapping around properly, and skipping disabled tabs